src/backend/loadContext.test.js

```diff
@@ -1,19 +1,46 @@
 // @flow
 
+import deepFreeze from "deep-freeze";
 import {type CacheProvider} from "./cache";
 import {type Project, createProject} from "../core/project";
+import {NodeAddress, EdgeAddress} from "../core/graph";
 import * as Weights from "../core/weights";
 import {validateToken} from "../plugins/github/token";
 import {TestTaskReporter} from "../util/taskReporter";
 import {LoadContext} from "./loadContext";
-
-const fakes = {
-  declarations: ({fake: "declarations"}: any),
+import {type PluginDeclaration} from "../analysis/pluginDeclaration";
+import {type NodeType} from "../analysis/types";
+
+const userType: NodeType = {
+  name: "user",
+  pluralName: "users",
+  prefix: NodeAddress.fromParts(["user"]),
+  defaultWeight: 0,
+  description: "a user",
+};
+const userPrefix = userType.prefix;
+const fooType: NodeType = {
+  name: "foo",
+  pluralName: "foos",
+  prefix: NodeAddress.fromParts(["foo"]),
+  defaultWeight: 0,
+  description: "a foo",
+};
+const plugin: PluginDeclaration = deepFreeze({
+  name: "foo",
+  nodePrefix: NodeAddress.empty,
+  edgePrefix: EdgeAddress.empty,
+  nodeTypes: [userType, fooType],
+  edgeTypes: [],
+  userTypes: [userType],
+});
```

Comment:
Likewise, having specific values instead of fakes, suggest we're doing logic.
LoadContext isn't supposed to know how to do these things, so it should be enough to be given fakes.